commons-extension-webapp/src/main/webapp/javascript/exo-i18n.js

```diff
@@ -29,6 +29,11 @@
   }
 
   function fetchLangFile(url, lang) {
+    if (url && url.indexOf('?') >= 0) {
+      url = `${url}&v=${eXo.env.client.assetsVersion}`
```

Comment:
Maybe I am wrong, but what if the URL is of pattern ?v=${eXo.env.client.assetsVersion}  already ? shouldn't we check that case ?

Reply:
This is not a problem. Having two `v=` will not lead to a problem since the parameter `v` will not be consumed/parsed by REST endpoint https://github.com/Meeds-io/commons/blob/develop/commons-component-common/src/main/java/org/exoplatform/commons/resource/ResourceBundleREST.java#L52
it's just to change the URI of the query so that the browser ignores its local cache and retrieve again the resource from server.